manifests/configmap.yaml

```diff
@@ -11,9 +11,10 @@ data:
 
   debug_logging: "true"
   workers: "4"
-  docker_image: registry.opensource.zalan.do/acid/demospilo-10:1.3-p3
+  docker_image: registry.opensource.zalan.do/acid/spilo-cdp-10:1.4-p8
+  pod_service_account_name: "zalando-postgres-operator"
   secret_name_template: '{username}.{cluster}.credentials'
-  # etcd_host: ""
+  etcd_host: ""
```

Comment:
is there a reason we need this line? could just remove it completely from the example?

Reply:
Currently it's required because the default for this `ConfigMap` field is nonempty](https://github.com/zalando-incubator/postgres-operator/blob/987b43456b080518c23a1242caa5c199ae7c1407/pkg/util/config/config.go#L72) and [that](https://github.com/zalando-incubator/postgres-operator/blob/c99cdd791584335203f5329e0c2f1b35fcfb938d/pkg/cluster/util.go#L464-L466) means [Spilo is set up to use etcd as DCS](https://github.com/zalando-incubator/postgres-operator/blob/e09e62f259a7bf5ba10099cbf2fb52a5ab2c5d57/pkg/cluster/k8sres.go#L402-L406) instead of the Kubernetes API.